expo/gbe/views/act_display_functions.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib import messages
+from django.core.urlresolvers import reverse
+from django.shortcuts import render
+from gbe.models import (
+    Act,
+    UserMessage
+)
+from gbetext import (
+    default_act_title_conflict,
+    act_not_unique
+)
+
+
+def display_invalid_act(request, data, form, conference, profile, view):
```

Comment:
It's avoiding common code in 2 views (create act, and edit act).  I actually tried to refactor those two views into 1 view (the django tutorial basically proposes that approach) but I got so many if/else cases built around the decision point of whether or not the act was new or edited - that the code became painful to read.

I eventually settled on this - a common function, so that common display points could share code.